docs/configuration.md

```diff
@@ -91,6 +92,17 @@ The availability zone names must be exposed as an environment variable `OPENSTAC
 
 The DNS servers must be exposed as an environment variable `OPENSTACK_DNS_NAMESERVERS`.
 
+## Machine flavor
+
+The flavors for control plane and worker node machines must be exposed as following environment variables 
```

Comment:
The flavors for control plane and worker node machines must be exposed as variables `OPENSTACK_CONTROL_PLANE_MACHINE_FLAVOR` and `OPENSTACK_NODE_MACHINE_FLAVOR` respectively.

Reply:
Done